astro/src/content/docs/get-started/core-concepts/integration-points.mdx

```diff
@@ -69,30 +70,36 @@ Each JWT has a header, a payload and a signature. Here's a diagram of a JWT:
 
 ### JWT Storage
 
-FusionAuth recommends the Authorization Code grant, where there is a server side component which exchanges the one-time use authorization code for an access token. This server side component offers a lot of flexibility when it comes to storing the JWT.
+FusionAuth recommends the Authorization Code grant, where there is a server-side component which exchanges the one-time use authorization code for an access token. This server side component offers a lot of flexibility when it comes to storing the JWT.
 
 But what should you do with it? After all, an access token is a bearer token, and should be properly secured. What you do with the token depends on what you are using it for as well as your security requirements. If you are wondering about your options, [here are a number of common login flows](/learn/expert-advice/authentication/login-authentication-workflows).
 
 Recommended storage mechanisms include:
 
 * sending the token down to the browser as a `Secure`, `HttpOnly` cookie. If you don't require cross-site cookie sharing, set `SameSite` to `Strict`. Otherwise, set `SameSite` to `Lax`, which will share the cookies in certain situations. Learn more about [`SameSite` settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite).
-* storing it server side, in a session. The session is typically managed by a framework, and ideally adheres to the same cookie storage recommendations. Learn more about [server side sessions](https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html). Please consult your framework documentation around securing sessions and data in sessions.
+* storing it server side, in a session. This is also known as the "backend for frontend" or BFF pattern. The session is typically managed by a framework, and ideally adheres to the same cookie storage recommendations. Learn more about [server-side sessions](https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html). Please consult your framework documentation around securing sessions and data in sessions.
+* sending the token down to a native app and storing in a secure storage area such as iOS Keychain or Android Keystore.
```

Comment:
```suggestion
* **_Native Secure Storage_** - sending the token down to a native app and storing in a secure storage area such as iOS Keychain or Android Keystore.
```

Reply:
I want to draw a strong visual line between the three bullets and the table headers... if I was visually scanning 👀 , I might miss what you were trying to do ......
<img width="970" alt="image" src="https://github.com/user-attachments/assets/165bf59f-f4df-4249-b7ca-dcd7ed0e6ef0" />